pages/api/contactForm.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  if (req.method === "POST") {
+    const { name, email, message } = req.body;
+
+    const webhookURL =
+      "https://discord.com/api/webhooks/1097847480697225226/8smtlEwEhZDyNFocN089-y2YFJqF34dAM7TLdP6X7nM3XeIC6qBLhMjO5bkCc34yWgeM";
+
+    const embed = {
+      title: "New message from a Portfolio Visitor",
+      fields: [
+        { name: "Name", value: name, inline: true },
+        { name: "Email", value: email, inline: true },
+        { name: "Message", value: message },
+      ],
+      timestamp: new Date().toISOString(),
+    };
+
+    try {
+      await axios.post(webhookURL, { embeds: [embed] });
```

Comment:
we never used axios in this course. You can use it of course, but it is an extra dependency that's not really needed for this one request. 

```suggestion
      await fetch(webhookURL, {
        method: "POST",
        headers: {
          "Content-Type": "application/json",
        },
        body: JSON.stringify({ embeds: [embed] }),
      });
```

You are not really doing anything with the response of the fetch (axios) request here - you might want to check if it was okay:

```suggestion
      const discordResponse = await fetch(webhookURL, {
        method: "POST",
        headers: {
          "Content-Type": "application/json",
        },
        body: JSON.stringify({ embeds: [embed] }),
      });
      if (!discordResponse.ok) throw new Error('Discord webhook fetch was not okay 🫣…') 
```
Or something like that ;)